configs/Pico/BoardConfig.h

```diff
@@ -184,7 +184,7 @@
 // 4 - `NOSPLASH` - This will not display a splash screen on boot
 // Special note - All of the splash screen images can be changed via `include/bitmaps.h`
 
-#define HAS_I2C_DISPLAY 1
+#define HAS_I2C_DISPLAY -1
```

Comment:
This is used for a bool, so it should be 0, not -1.

Reply:
There are a couple other BoardConfig.h files that make this mistake, and even `headers/addons/i2cdisplay.h` makes the mistake. I think we should fix `i2cdisplay.h` and remove all board configs that have it defined to 0 (or -1 erroneously) for sake of simplicity.

Reply:
Thanks - just saw the comment pop up here :). Fixed!  

We should go through and check the other configs for this